samples/cpp/essential_mat_reconstr.cpp

```diff
@@ -0,0 +1,345 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include "opencv2/calib3d.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+
+#include <vector>
+#include <iostream>
+#include <fstream>
+
+using namespace cv;
+double getError2EpipLines (const Mat &F, const Mat &pts1, const Mat &pts2, const Mat &mask);
```

Comment:
instead of adding prototype for each function, just make all the functions, except 'main' static

Reply:
fixed